schema.go

```diff
@@ -1,7 +1,28 @@
 package schemas
 
-import "embed"
-
-//go:embed schemas/openapi.yml
+import (
```

Comment:
Is this appropriate for the schemas repo or more so for meshkit repo?

Other golang components that use these schemas will import the schemas as packages and have full access to the content of each schema. Is that right?

I'm trying to understand the need for this utility function.

Reply:
Yes, this will allow other components to have full access to the content of each schema